### PR TITLE
Add Chronicle backend

### PR DESCRIPTION
**Pull request by kylesmartin**

### CODE REVIEW

File: fig/worker.py [hunk 1]
```diff
@@ -23,6 +23,4 @@ def run(self):
 
     def process_event(self, event):
         falcon_event = FalconEvent(event, self.cache)
-        if falcon_event.cloud_provider is None and falcon_event.mdm_identifier is None:
-            return
```
Comment (isimluk) [inline fig/worker.py:27]: Oh, forgot this one. Thanks for removal.

**Comment by redhatrises** [comment]
Thanks for the PR! Looks like there are some linting issues that need to be fixed. BTW, `autopep8` can sometimes be your friend.

**Review by isimluk — APPROVED** [review]
👍

**Comment by redhatrises** [comment]
Thanks for the fixes and code enhancements!

**Pull request merged**